projects/remult-admin/src/App.tsx

```diff
@@ -35,46 +35,57 @@ function App() {
   return (
     <>
       <BrowserRouter basename={options?.baseUrl}>
-        <div>
-          <NavLink className="tab" to="erd">
-            ERD
-          </NavLink>
-          {god?.tables.map((t) => (
-            <NavLink className="tab" key={t.key} to={t.key}>
-              {t.caption}
+        <div class="app-holder">
```

Comment:
tsx doesn't like class - it should be 'className'
```suggestion
        <div className="app-holder">
```